apps/dotcom/src/utils/iFrame.ts

```diff
@@ -0,0 +1,20 @@
+export const isInIframe = () => {
+	return typeof window !== 'undefined' && (window !== window.top || window.self !== window.parent)
+}
+
+export function getTopLevelOrigin() {
```

Comment:
i think we should call this `getParentOrigin` and use `ancestorOrigins[0]` instead because:
- that is a bit more consistent with what `document.referrer` might do
- the parent is more truly the embedder that the top level document, I would guess? e.g. if i embed tldraw in notion then embed notion in my website, notion is still the main embedder i think?

what do you reckon?

Reply:
Definitely 👍